inference-engine/src/transformations/src/transformations/op_conversions/convert_broadcast_to_tiles.cpp

```diff
@@ -23,17 +23,21 @@ ngraph::pass::ConvertBroadcastToTiles::ConvertBroadcastToTiles() {
             return false;
         }
 
-        auto data_node = broadcast->input_value(0).get_node_shared_ptr();
+        auto data_node = broadcast->input_value(0);
+        if (data_node.get_partial_shape().is_dynamic()) {
```

Comment:
Why don't we use `pattern::any_input(pattern::has_static_shape()),` for this case?

Reply:
Broadcast has multiple inputs and there is no way to specify optional input with predicate. I'm working on that but for now I would keep this check here.